torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -198,6 +202,102 @@ bool is_numpy_scalar(PyObject* obj) {
           PyArray_IsScalar(obj, Floating));
 }
 
+at::Tensor tensor_from_cuda_array_interface(PyObject* obj) {
+  PyObject *cuda_dict = PyObject_GetAttrString(obj, "__cuda_array_interface__");
+  if (cuda_dict == nullptr) {
+    throw TypeError("attribute `__cuda_array_interface__` must exist");
```

Comment:
This isn't just a TypeError; if this ever fails, that's an internal error (the precondition for this function is that `__cuda_array_interface__` exists, right?) So a `TORCH_INTERNAL_ASSERT` is an appropriate thing to use to check the nullptr here.

Reply:
Fixed in 5b0c83c